ThreadExample/src/ThreadExercise.java

```diff
@@ -0,0 +1,11 @@
+public class ThreadExercise implements Runnable{
+    @Override
+    public void run() {
+        System.out.println("I am a running thread");
+        try {
+            Thread.sleep(5000);
```

Comment:
@maria-salmasian each thread need to run by random period

Reply:
changed it